Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -46,68 +73,147 @@ class IntroViewController: UIViewController, OnViewDismissable {
     }
 
     // MARK: View setup
-    private func initialViewSetup() {
-        setupIntroView()
-    }
-
-    // onboarding intro view
-    private func setupIntroView() {
-        // Initialize
-        view.addSubview(syncCard)
-        view.addSubview(welcomeCard)
+    private func setupPageController() {
+        // Create onboarding card views
+        var cardViewController: OnboardingCardViewController
+        for (index, cardType) in viewModel.enabledCards.enumerated() {
+            if cardType == .wallpapers {
+                cardViewController = WallpaperCardViewController(viewModel: viewModel.getCardViewModel(index: index),
+                                                                      delegate: self)
+            } else {
+                cardViewController = OnboardingCardViewController(viewModel: viewModel.getCardViewModel(index: index),
+                                                                      delegate: self)
+            }
+            onboardingCards.append(cardViewController)
```

Comment:
Indentation seems off on XCode and github 

[Before]

<img width="400" alt="image" src="https://user-images.githubusercontent.com/8919439/172297627-d6689492-cbfc-41cd-8dab-002c383ae8b3.png">

This is the best I feel we can fix it [after] 

<img width="974" alt="image" src="https://user-images.githubusercontent.com/8919439/172297886-25602ac9-773f-4418-b19d-747f31bdad06.png">